login_server/player/state/player_state.py

```diff
@@ -69,7 +69,9 @@ class PlayerState:
     def __init__(self, player: Player):
         self.logger = logging.getLogger(__name__)
         self.player = player
-        self.class_menu_data = get_class_menu_data(self.player.use_goty_mode)
+
+    def class_menu_data(self):
```

Comment:
I think this should be a member of player and not player_state, so it won't have to touch player's game_setting_mode from the outside.

Reply:
This is outdated, but I've made a change to the new code anyway, since we now only ever need to send OOTB data over the main login protocol.